docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -425,6 +425,21 @@ URL testUrl;
 The test port can be controlled via the `quarkus.http.test-port` config property. Quarkus also creates a system
 property called `test.url` that is set to the base test URL for situations where you cannot use injection.
 
+== Working with multi-module project or external modules
+
+Quarkus uses Jandex to create, at build time, an index of all classes eligible to CDI bean discovery.
+Without it, none of the Quarkus extensions will work.
+
+This index is created by default on your project thanks to our Maven or Gradle plugin.
```

Comment:
I am not sure that's accurate. Maven/gradle indexing plugins are configured by users in their projects if they choose to do that. We are indexing certain dependencies at build time (unless the index is already available).

Reply:
We automatically index _application classes_, that wasI refers to. If it's not via the Maven/Gradle plugin I can remove this part of the sentence.
But this means that you have nothin to do on your Quarkus projet ...

Reply:
Ok, I probably misunderstood what you wanted to say here. My point was that it was not the Maven/plugins themselves that do that but given that they do launch the Quarkus build process which also includes the indexing phase, that's probably ok.

Reply:
Maybe we can find a better way to express this. It is Quarkus that creates the index on the project that bootstrap it, difficult to express this in a way that everybody understood.

Reply:
Ok, how about this:

Quarkus uses Jandex to create, at build time, an index of all the classes eligible to CDI bean discovery. The application project classes will be indexed by Quarkus by default.